catchup/service.go

```diff
@@ -575,7 +583,12 @@ func (s *Service) periodicSync() {
 				// keep the existing sleep duration and try again later.
 				continue
 			}
-			s.suspendForCatchpointWriting = false
+			// if the ledger has too many non-flushed account changes, skip
+			if !s.ledger.IsBehindCommittingDeltas() {
```

Comment:
"IsBehind" means it's _not_ available, right? So the condition has to be reversed.

Reply:
Fixed, thanks